### PR TITLE
Issue 23 experts grid component

### PR DESCRIPTION
**Pull request by aleksandra04**
develop

implemented css grid

## GitHub Board

**Issue link**

[#23 experts grid component ]( )

**Story link**

[#13 Experts]()

## Code reviewers
- [ ] @github_username

## Summary of issue

- [ ]  Summary of issue
- [ ]  Summary of issue

## Summary of change

- [ ]  Summary of change
- [ ]  Summary of change

### CODE REVIEW

File: src/modules/experts/styles/ExpertBlock.style.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  },
+  item_8: {
+    gridArea: '1 / 5 / 3 / 6',
+  },
+  item_9: {
+    gridArea: '3 / 5 / 5 / 6',
+  },
+  item_10: {
+    gridArea: '1 / 6 / 5 / 7',
+  },
```
Comment (SergeOlabin) [inline src/modules/experts/styles/ExpertBlock.style.ts:36]: these should not be set in the `ExpertBlock`.

pass the styles from parent grid component, it should be responsible for styling it's children.
item doesn't have to know about it's position, it has to display the content, and that's it.
parent should be responsible for positioning its items.

File: src/lib/components/ExpertBlock.tsx [hunk 2]
```diff
@@ -1,15 +1,24 @@
 import React from 'react';
+import { useStyles } from '../../modules/experts/styles/ExpertBlock.style';
 import { IExpert } from '../types';
 
 interface IExpertProps {
   expert: IExpert;
+  nameClass: string;
```
Comment (SergeOlabin) [inline src/lib/components/ExpertBlock.tsx:7]: see another comment, it should not be here

**Review by SergeOlabin — CHANGES_REQUESTED** [review]

File: src/modules/experts/styles/ExpertBlock.style.ts (hunk 1 above)
Comment (aleksandra04) [reply src/modules/experts/styles/ExpertBlock.style.ts]: moved to MainExpertsView block

File: src/lib/components/ExpertBlock.tsx (hunk 2 above)
Comment (aleksandra04) [reply src/lib/components/ExpertBlock.tsx]: deleted

**Review by SergeOlabin — APPROVED** [review]

**Pull request merged**